diff -rupN grub-1.99.orig//configure grub-1.99//configure
--- grub-1.99.orig//configure	2011-05-14 20:57:49.000000000 +0000
+++ grub-1.99//configure	2011-08-03 13:18:05.013238567 +0000
@@ -3821,7 +3821,7 @@ TARGET_CPPFLAGS="$TARGET_CPPFLAGS -I\$(t
 TARGET_CPPFLAGS="$TARGET_CPPFLAGS -I\$(top_builddir)/include"
 
 case "$target_cpu" in
-  i[3456]86)	target_cpu=i386 ;;
+  i[3456]86|x86)	target_cpu=i386 ;;
   amd64)	target_cpu=x86_64 ;;
   sparc)	target_cpu=sparc64 ;;
   mipsel|mips64el)
diff -rupN grub-1.99.orig//configure.ac grub-1.99//configure.ac
--- grub-1.99.orig//configure.ac	2011-05-13 19:47:26.000000000 +0000
+++ grub-1.99//configure.ac	2011-08-03 13:14:35.012238732 +0000
@@ -68,7 +68,7 @@ TARGET_CPPFLAGS="$TARGET_CPPFLAGS -I\$(t
 TARGET_CPPFLAGS="$TARGET_CPPFLAGS -I\$(top_builddir)/include"
 
 case "$target_cpu" in
-  i[[3456]]86)	target_cpu=i386 ;;
+  i[[3456]]86|x86)	target_cpu=i386 ;;
   amd64)	target_cpu=x86_64 ;;
   sparc)	target_cpu=sparc64 ;;
   mipsel|mips64el)
